docs/getting_started/tutorial.md

```diff
@@ -349,6 +349,8 @@ other page type.
 
 ![](../_static/images/tutorial/tutorial_5.png)
 
+Be sure to *Publish* the blog post after you are done.
```

Comment:
How about "*Publish* each blog post after you are done editing."? (Just because there's not just a single blog post). But other than that, looks great!